python/typemap_utils.cpp

```diff
@@ -25,7 +25,7 @@
     #define PyInteger_FromLong(n) PyLong_FromLong(n)
 #else
     #define PyObject_ToCharPtr(n) PyString_AsString(n)
```

Comment:
Note that `PyString_AsString` "computes the default encoding of string and operates on that" according to the manual, whereas we probably should use UTF-8 (which will work on everything except Windows, and on Windows will work for strings containing only ASCII characters)?

To do this, you might need to call `PyUnicode_AsUTF8String` on Unicode strings to convert them to strings in the utf-8 encoding, and then call `PyString_AsString` on the result.